curriculum/Chapter-2-Agile-Team-Collaboration.md

```diff
@@ -0,0 +1,140 @@
+- Master the planning processes and point systems that drive workload management and prioritization.
+- Learn the nuances of sprints, from pre-planning to demonstrations, and the importance of collecting user feedback.
+
+## Introduction to Agile
+
+Agile is a modern approach to software development that enables teams to deliver value to their customers effectively and efficiently. This methodology is underpinned by a set of principles that prioritize flexibility, collaboration, and the continuous delivery of functional software. At its core, Agile is not a prescriptive framework with rigid rules but rather a philosophy that guides decision-making processes. It encourages teams to view the development cycle as a series of iterative releases, allowing for frequent reassessment of priorities and incorporation of feedback to better meet customer needs.
+
+In this chapter, we'll explore how Agile principles apply in the context of team collaboration and project management. We'll dissect how the principles and practices of Agile can not only improve the quality of software but also enhance the productivity and engagement of Agile teams. For further reading, we have linked the [Agile Manifesto](https://agilemanifesto.org/principles.html) as well
+
+### Rise8 Agile Principles
+
+TODO: Elaborate on the Rise8 Agile Principles that tailor the Agile methodology to our specific organizational context and goals.
+
+## Pair Programming
+
+Pair programming is a development technique where two developers author software using the same computer. In a remote environment, one user can share the screen with another via collaboration software such as Zoom or Live Share. There are two roles in pair programming:
+
+- Driver: The person who is writing the code.
+- Navigator: The person who helps the driver navigate the code development process. They can write code in the form of suggestions or corrections.
+
+Here are a few helpful hints when pairing:
+
+- Pairing can be tiring, take breaks often.
+- Rotate pairs regularly. Each person brings something unique to the table which will improve the codebase as a whole. Swapping pairs also drives both knowledge sharing and alignment across the team.
+- Be open to new ideas and constructive criticism.
+- Sometimes pairing might not be the best approach. Feel free to solo when it makes sense. But remember, committed code requires a peer review.
+
+## Roles and Responsibilities
+
+### Balanced Teams
+
+Within the agile framework, the concept of balanced teams has emerged as a cornerstone for providing value to the end users. A balanced team, typically composed of engineers, designers, and product managers, embodies a holistic approach to product development, where collaboration across disciplines is not just encouraged — it's a fundamental principle. This section explores how balanced teams foster collaboration between different disciplines to create better product outcomes and, ultimately, deliver exceptional value to users.
```

Comment:
Maybe bring up why Rise8 doesn't encourage QA teams, or other forms of silo'd teams - where tighter feedback loops and context spreading and awareness is an advantage of having a balanced team.